e2e/suite1/specs/panelEdit_base.spec.ts

```diff
@@ -38,15 +39,17 @@ e2e.scenario({
         e2e.components.Transforms.card('Merge').scrollIntoView().should('be.visible');
         e2e.components.QueryTab.content().should('not.exist');
         e2e.components.AlertTab.content().should('not.exist');
+        e2e.components.PanelAlertTabContent.content().should('not.exist');
 
         //  Can change to Alerts tab (graph panel is the default vis so the alerts tab should be rendered)
         e2e.components.Tab.title('Alert').should('be.visible').click();
         e2e.components.Tab.active().within((li: JQuery<HTMLLIElement>) => {
-          expect(li.text()).equals('Alert0'); // there's no alert so therefore Alert + 0
+          expect(li.text()).equals('Alert'); // there's no alert so therefore Alert
```

Comment:
not sure about this; I don't understand which is this list item but it seems to fixes the failure.